JS/Lesson16/sinan/script.js

```diff
@@ -1,142 +1,177 @@
 // 1. Write a loop that prints out the numbers from 1 to 10.
-for (let i = 1; i<=10; i++) {
+for (let i =0; i<10; i++) {
 	console.log(i);
-}
+	}
 console.log('- - - - - - - - - - - - - - - - - ');
 // 2. Write a loop that prints out the odd numbers from 1 to 20.
 // 1,3,5,7...
-for (let i = 1; i<20; i = i+2){
+for(let i=1; i<20; i+=2){
 	console.log(i);
 }
 
 console.log('- - - - - - - - - - - - - - - - - ');
 // 3. Write a loop that prints out the even numbers from 1 to 20.
 // 2, 4, 6...
-for (let i =2; i<=20; i+=2) {
+for(let i =0; i<20; i+=2) {
```

Comment:
0 is not an even number, should be like this instead:
```suggestion
for(let i =2; i<=20; i+=2) {
```